packages/entities/entities-plugins/fixtures/mockData.ts

```diff
@@ -390,6 +390,30 @@ export const kmAvailablePlugins = {
         version: '3.6.0',
         priority: 400,
       },
+      "ai-response-transformer": {
```

Comment:
Could you change these double quotes to single quotes?
```suggestion
      'ai-response-transformer': {
```